index.html

```diff
@@ -43,6 +43,153 @@ <h3>LET'S CONNECT</h3>
+        <div class="project">
+            <img class="project_image" src="assets/Snapshoot Portfolio (2).png" alt="">
+            <div class="general">
+                <h3>Tonic</h3>
+                <div class="general_inf">
+                    <p>Canopy.Back-end.2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+        </div>
+        <div class="project">
+            <img class="project_image" src="assets/Snapshoot Portfolio (3).png" alt="">
+            <div class="general">
+                <h3>Multi-post Stories</h3>
+                <div class="general_inf">
+                    <p>Canopy.Back-end.2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+        </div>
+        
+
+    </div>
+    <section class="last-sect">
+        <section id="about-myself">
+        <div class="myself-abt">
+            <h1>About<br>Myself</h1>
+            <p id="presentation">I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                on’t hestiate to contact me.
+            </p>
+            <div id="resume-link">
+                <h2>LETS CONNECT</h2>
+                <ul>
+                    <li><a href="https://www.linkedin.com/in/venkatesh-shri-sakthi/" ><img src="assets/Linkedin icon.png" ></a></li>
+                    <li><a href="https://github.com/shri-sakthi-v"><img src="assets/Vector (1).png" ></a></li>
+                    <li><a href="https://twitter.com/shrisakthi"><img src="assets/Vector.png" ></a></li>
+                    <li><a href="https://twitter.com/shrisakthi"><img src="assets/storm.png" alt="#"></a></li>
+                    <li><a href="https://twitter.com/shrisakthi"><img src="assets/mail.png" alt=""></a></li>
+                </ul>
+                <a href="#" id="get-resume">Get my resume</a>
```

Comment:
Please also update this button as well🙏

Reply:
thank you for notifying me, but right now my resume template is not outlined by the microverse team.
the resume template is not mentioned in the daily progress for the first week.